Add rrfs.yaml

maddenp-cu opened the pull request:
**Synopsis**

Addresses [this issue](https://github.com/orgs/NOAA-GSL/projects/43/views/3?pane=issue&itemId=107978310&issue=NOAA-GSL%7Cmpas_planning%7C45) by adding an `rrfs.yaml` UW YAML for Rocoto workflow file.

**Type**

- [x] Enhancement (adds new functionality)

**Impact**

- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

christinaholtNOAA reviewed (commented):
This is great! 

As I mentioned in standup, we want to consider what we do with this file. It might be good either include it as a dot-file, or to add a dot-directory where we keep some of these in-progress pieces. We could move toward using mpas_app scripts and drivers to configure some of these components (mpas_init, forecast, upp, etc.).

maddenp-cu commented on ush/workflows/.incubating/rrfs.yaml:
```diff
@@ -0,0 +1,520 @@
+      nodes: 1:ppn=1
+      partition: hera
+      queue: batch
+      walltime: "1:30:00"
```
The name of this task sounds like it's copying data or something like that. If so, I'm not sure why it'd need 1h30m. It appears to be a serial process based on `nodes: 1:ppn=1`.

christinaholtNOAA replied:
I think it's running alongside the forecast. I see that there's a [loop in the script waiting](https://github.com/NOAA-EMC/rrfs-workflow/blob/ca30e960ce92b5c449b38ec7dc17b27043d42295/scripts/exrrfs_save_fcst.sh#L36-L42) for the next file to be ready.

maddenp-cu commented:
I made a couple updates:

- Replaced some of the complex dependency blocks with simpler `sh:`-based ones. I tested the commands used there in `bash` with synthetic test data and they seemed correct, but we'll have to find out for sure later when we're in a position to fully test this workflow.
- Removed all the ostensibly pointless constructs.
- Moved `rrfs.yaml` to a new `.incubating/` directory, to discourage its use by users for now.

I think this is worth another quick look.

christinaholtNOAA commented on ush/workflows/.incubating/rrfs.yaml:
```diff
@@ -0,0 +1,522 @@
+              cyclestr:
+                attrs:
+                  offset: "-6:00:00"
+                value: "test $(find {{ common.retro_gfs }} -mmin +5 -regex '.*/gfs\\.t@Hz\\.pgrb2[b]?\\.0p25\\.f006' | wc -l) -eq 2"
```
🎉

christinaholtNOAA reviewed (approved):
Nice clean up! Thanks for moving the file and for squashing some of that logic.
